src/komponenter/banner/banner.tsx

```diff
@@ -44,6 +36,15 @@ class Banner extends React.Component<Props> {
         return !(pathname.includes(START_PATH)
             && (this.props.startRegistreringStatus.underOppfolging === false));
     }
+
+    settBannerOverskriftId(): string {
```

Comment:
Jeg mener det er en "best practice" som sier at `render()` bør ligge nederst i komponenten, men det er mulig jeg er utdatert 😅 Er det en spesiell grunn til at du la denne funksjonen under `render()`?

Reply:
Nei, du har helt rett - men jeg så skalVises lå inni render() og tenkte kanskje det var noe typescript greier jeg ikke var klar over og valgte å gjøre det på samme måten, Jeg kan flytte begge ut av render() hvis du ønsker det.

Reply:
Jeg tenker det fint kan ligge som funksjoner inne i `render` hvis funksjonen ikke eksplisitt benytter seg av `this`

Reply:
Har flyttet begge funsjonene og bannerId ut av render. Nå settes bannerOverskrifId kun en gang og ikke hver rerender som tidligere.